components/provisioner/internal/api/resolver_integration_with_gardener_test.go

```diff
@@ -559,9 +552,10 @@ func testProvisioningTimeouts() queue.ProvisioningTimeouts {
 
 func testProvisioningNoInstallTimeouts() queue.ProvisioningNoInstallTimeouts {
 	return queue.ProvisioningNoInstallTimeouts{
-		ClusterCreation:  5 * time.Minute,
-		ClusterDomains:   5 * time.Minute,
-		BindingsCreation: 5 * time.Minute,
+		ClusterCreation:    5 * time.Minute,
+		ClusterDomains:     5 * time.Minute,
+		BindingsCreation:   5 * time.Minute,
+		AgentConfiguration: 5 * time.Minute,
```

Comment:
magic value

Maybe the time could be extracted to a const?
Not important - I'll approve despite this